system-test/fixtures/sample/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "./node_modules/gts/tsconfig-google.json",
   "compilerOptions": {
+    "lib": ["es2015", "dom"],
```

Comment:
I suspect this may be a very bad idea.  I had to add `dom` to my list of libs here to ensure the `window` type could compile.  As a result of adding `nodeFetch | window.fetch` in the types, I had to ensure `window` was specifically discoverable via TypeScript.  I'm worried this will break every TypeScript consumer of ... well every library we ship.